auto3dseg/algorithm_templates/dints/scripts/infer.py

```diff
@@ -72,8 +72,10 @@ def pre_operation(config_file, **override):
 
     for _file in config_file:
         if _file.endswith("hyper_parameters.yaml"):
-            parser = ConfigParser(globals=False)
-            parser.read_config(_file)
+            lock = FileLock(f"{_file}.lock")
```

Comment:
Is this .lock file going to be deleted?

Reply:
I think we can leave it here? The lock status is get from `with lock:` block.